autodoc/admin-api/data/admin-api.lua

```diff
@@ -810,8 +810,8 @@ return {
         },
         protocols = {
           description = [[
-            A list of the protocols this Route should allow. When set to `["https"]`,
-            HTTP requests are answered with a request to upgrade to HTTPS.
+            An array of the protocols this Route should allow. To allow only one protocol, send a string value. See the Route Object section for a list of accepted protocols. When set to only `"https"`,
```

Comment:
```suggestion
            An array of the protocols this Route should allow. To allow only one protocol, send a string value. See the [Route Object](#route-object) section for a list of accepted protocols. When set to only `"https"`,
```
If this link formatting is fine to use in this file, better to link the to top of the section if possible.